demo/completer-cmp-md.html

```diff
@@ -1,6 +1,6 @@
 <div class="completer-holder" ctrCompleter>
     <md-input-container class="completer-input">
-        <input mdInput ctrInput="clearSelected=clearSelected; overrideSuggested=overrideSuggested; fillHighlighted=fillHighlighted" [(ngModel)]="searchStr" 
+        <input mdInput ctrInput="clearSelected=clearSelected; overrideSuggested=overrideSuggested; fillHighlighted=fillHighlighted; forceSelection=forceSelection" [(ngModel)]="searchStr" 
```

Comment:
I think it'll be better to change one of the `nativeCmp` demos to show how it's working